src/encoded/static/components/body_map.js

```diff
@@ -126,6 +126,34 @@ const addingClass = (changedClass, matchingString, removeFlag = false) => {
     }
 };
 
+// Set body map diagram colors based on url, when component mounts
+export const initializeBodyMap = (searchQuery) => {
+    const query = new QueryString(searchQuery);
+    const terms = query.getKeyValues(organField);
+    if (terms.length > 0) {
+        terms.forEach((term) => {
+            if (BodyList[term]) {
+                BodyList[term].forEach((bodyClass) => {
+                    addingClass('active', bodyClass);
+                });
+            }
+            if (SystemsList[term]) {
+                document.getElementById(term).classList.add('active');
+            }
+        });
+    }
+};
+
+// Clearing organ and systems key value selections from a url
+export const clearBodyMapSelectionsFromUrl = (originalUrl) => {
```

Comment:
"clearBodyMapSelectionsFromUrl " or "removeSystemsFromUrl"?

Reply:
I think that keeping this as "clearBodyMapSelectionsFromUrl" is the most clear, because the function clears both systems and organ fields from the url and "body map selections" encapsulates both.